src/main/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspection.java

```diff
@@ -45,6 +46,7 @@ public class WrongUsageOfMappersFactoryInspection extends InspectionBase {
         MapstructUtil.MAPPERS_FQN,
         "getMapper"
     ).parameterTypes( CommonClassNames.JAVA_LANG_CLASS );
+    public static final String COMPONENT_MODEL = "componentModel";
```

Comment:
Does this need to be `public`?

Reply:
No doesn't need to be. Change to `private`.